pep-0008.txt

```diff
@@ -241,6 +241,17 @@ Another such case is with ``assert`` statements.
 
 Make sure to indent the continued line appropriately.
 
+
+Exception Message String Literals
+---------------------------------
+
+Strive to keep exception message string literals on one line,
+in order to make them easily greppable.
+If your exception message exceeds the maximum line length,
```

Comment:
This could be a link, but equally it follows the line length section directly

Reply:
I'm not sure it's needed, but I'm also fine with a link. I'll leave it for you and Christopher to decide 🙂

(IIRC, both of you have the commit bit here, so feel free to apply the suggestion when you've decided.)

Reply:
> I'm not sure it's needed, but I'm also fine with a link. 

I'm not sure it is either, since it is a short section and immediately above. But I don't have a strong opinion.

> I'll leave it for you and Christopher to decide slightly_smiling_face

You mean @Rosuav ? Or are you referring to me (CAM)?

Reply:
I was referring to you, CAM :)

Reply:
> I was unaware that there was more than two PEP editors

Guido, Barry, Bret, ChrisA, Jelle, Adam, Cam, and Hugo are the current complement -- a few more than two!

A

Reply:
It's okay, most of us are invisible :) For what it's worth, I have no opinion on the specific wording, but I fully approve of the sentiment here.

Reply:
Can I mark this as resolved? Did you decide about the link?